ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -57,9 +58,8 @@ def _before_flask_request():
     span_name = flask_request.endpoint or otel_wsgi.get_default_span_name(
         environ
     )
-    parent_span = propagators.extract(
-        otel_wsgi.get_header_from_environ, environ
-    )
+    context = propagators.extract(otel_wsgi.get_header_from_environ, environ)
+    parent_span = get_span_from_context(context).get_context()
```

Comment:
I think we may need to add another line here to extract or pass through correlation context, since we're not attaching this context as we start the activation.

Would it make sense to add context attach / detach to these extensions?

Reply:
i think so. added an attach/detach to flask and wsgi extensions.